src/extension.ts

```diff
@@ -84,4 +85,17 @@ function getDocsets(languageId: string): Array<string> {
     }
 
     return [];
-}
\ No newline at end of file
+}
+
+/**
+ * Handle weird Zeal bug where it does not start up correctly
+ *
+ * @param {Dash} dash dash object
+ * @return {void}
+ */
+function handleZeal(dash: Dash): void {
+    if (dash.usingZeal()) {
+        exec('zeal');
+        // not sure why but the other command does not open zeal correctly sometimes, there's an error in Qt

```

Comment:
I'm not entirely sure, but it seems to be Windows-specific. The first Zeal startup cannot handle the Dash URI as an argument from node. So this just starts Zeal first before calling it on the URI. Works fine on Linux though.